docs/usage.rst

```diff
@@ -74,6 +76,39 @@ with the fake file system functions and modules:
 The usage is explained in more detail in :ref:`auto_patch` and
 demonstrated in the files `example.py`_ and `example_test.py`_.
 
+If your setup is the same for all tests in a class, you can use the class setup
+method ``setUpClassPyfakefs`` instead:
+
+.. code:: python
+
+    from pyfakefs.fake_filesystem_unittest import TestCase
+
+
+    class ExampleTestCase(TestCase):
+        def setUpClass(cls):
```

Comment:
Shouldn't there be an `@classmethod` as decorator?

Reply:
Yes, looks like I botched the example... thanks, will correct this!